sonarqube-companion-rest/src/test/java/pl/consdata/ico/sqcompanion/BaseItTest.java

```diff
@@ -37,7 +37,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(SpringRunner.class)
-@SpringBootTest(classes = {BaseItTest.ItTestConfiguration.class})
+@SpringBootTest(classes = {BaseItTest.ItTestConfiguration.class}, properties = "spring.main.allow-bean-definition-overriding=true")
```

Comment:
po co to?

Reply:
Od 2.1 to jest domyślnie na false, przez co context testów nie wstaje.
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding

Reply:
W testach nadpisujemy beana fasady i configuracji